db/controller.js

```diff
@@ -48,4 +48,20 @@ const getAllArticles = (req, res, next) => {
     })
 }
 
-module.exports = {getAllTopics, getAllEndpoints, getArticleById, getAllArticles}
\ No newline at end of file
+const getCommentsById = (req, res, next) => {
+    
+    const { article_id } = req.params;
+    if ( isNaN(article_id)) {
```

Comment:
PSQL would handle this if you have written the appropriate error handler for a 22P02 error